spec/features/spree/admin/reporting_to_taxjar_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.feature 'Reporting orders to TaxJar', js: true, vcr: true do
+  stub_authorization!
+
+  background do
+    create :store, default: true
+    create :taxjar_configuration, :reporting_enabled
+  end
+
+  let!(:order) { create :order_ready_to_ship }
+
+  scenario "shipping a complete and paid order", vcr: { allow_unused_http_interactions: false } do
+    visit spree.edit_admin_order_path(order)
+    perform_enqueued_jobs do
+      expect(page).not_to have_content("Loading")
```

Comment:
(Maybe ignore this comment. I am nitpicking.)

As a Rails developer, I understand what is implied by this expectation and why we might want to expect it. But I think the way it's written doesn't really map to something a user would do. (I.e. users don't wait for a page to stop saying loading. They wait for a page to finish loading.)

Maybe this is a bad suggestion, but I might write this this way to make the intent more clear:

```
  # ...
  visit spree.edit_admin_order_path(order)
  perform_enqueued_jobs do
    wait_for_order_information_to_load
    # ...
  end
end
    
def wait_for_order_information_to_load
  expect(page).not_to have_content "Loading"
end
```

Reply:
It's a good nit! Thanks for the reminder as I haven't been doing this in my tests lately.

It does make the intent more clear

Reply:
Updated!